lib/internal/backend/file/file.go

```diff
@@ -260,7 +276,7 @@ func getImageID(file *os.File, dockerURL *common.ParsedDockerURL, name string, d
 
 			if dockerURL == nil {
 				dockerURL = &common.ParsedDockerURL{
-					OriginalName: "",
+					OriginalName: appName + ":" + tag,
```

Comment:
Does this break for images saved from `foo@sha256:$hash`? Can you even save those without tagging them to some string?